website/docs/components/dropdown/index.md

```diff
@@ -8,19 +8,20 @@ links:
   github: https://github.com/hashicorp/design-system/tree/main/packages/components/addon/components/hds/dropdown
 ---
 
-<section data-tab="Other">
-  @include "partials/other/generic-1.md"
-  @include "partials/other/generic-2.md"
+<section data-tab="Guidelines">
+  @include "partials/guidelines/overview.md"
+  @include "partials/guidelines/guidelines.md"
 </section>
 
 <section data-tab="Code">
-  @include "partials/code/component-api.md"
   @include "partials/code/how-to-use.md"
+  @include "partials/code/component-api.md"
   @include "partials/code/showcase.md"
```

Comment:
the showcase file needs updating to add `## Showcase` to the top of it so it'll appear in the sidecar